events/events.go

```diff
@@ -196,3 +204,20 @@ func (e *Events) unsubscribe(l listener) bool {
 	e.mu.Unlock()
 	return false
 }
+
+func GetContainerId(eng *engine.Engine, name string) string {
+	var buf bytes.Buffer
+	job := eng.Job("container_inspect", name)
+
+	var outStream io.Writer
+
+	outStream = &buf
+	job.Stdout.Set(outStream)
+
+	if err := job.Run(); err != nil {
+		return ""
+	}
+	var out struct{ ID string }
+	json.NewDecoder(bytes.NewReader(buf.Bytes())).Decode(&out)
```

Comment:
@brahmaroutu Oh right buf is not a pointer :) So just do `json.NewDecoder(&buf)` :)